tests/test_transitions.py

```diff
@@ -0,0 +1,79 @@
+from selenium.webdriver.common.by import By
+from selenium.webdriver.remote.webdriver import WebDriver
+from selenium.webdriver.common.action_chains import ActionChains
+
+MAIN_PAGE = "https://machinist.flapflap.io/"
+
+
+def test_transitioning_nodes(driver: WebDriver):
```

Comment:
Hey @amneet270, what do you think about testing two way transition? Like here we are testing a single transition on the page, an improvement to the test would be to test if a transition could also be made in the reverse direction.

Also, testing creating transitions when there are more nodes on the canvas too. Like 3 nodes.

Reply:
I was going to do that as a different task but you are right it make more sense to directly do it here I will make the required changes